api/src/main/java/org/eclipse/microprofile/metrics/MetricID.java

```diff
@@ -70,6 +70,9 @@
 
     public static final String GLOBAL_TAGS_VARIABLE = "mp.metrics.tags";
 
+    public static final String APPLICATION_NAME_VARIABLE = "mp.metrics.appName";
+    public static final String APPLICATION_NAME_TAG = "_app";
```

Comment:
More clarifying than anything else: did we settle on `_app` and not e.g. `mpApp` or `_mpApp`?

Reply:
Hmm. At least in the meeting minutes from 25 March it is `_app` :)